### ISSUE TITLE
Video captions don't work

### ISSUE THREAD

**Issue by gurupras** [Selz/plyr#162]
I've been trying to get video captions to work for a while and they've been consistently failing.
Unfortunately, since I don't understand much of the source code that goes into rendering captions, I'm unable to track the issue down further.

However, to help pinpoint the issue, I have created a test page should help in reproducing the bug.

I've written [this](https://twoseven.xyz/test-captions) page that demonstrates the problem.
As seen on that page, on Chrome, the native player renders captions correctly whereas plyr does not.

**Comment by SamPotts** [comment]
Sounds like I need to have another look at how the captions are working. The support was mixed at the start of the project and I think things have changed recently. I'll take a look.

**Comment by SamPotts** [comment]
Thanks for your fix, can we close this one now?

**Comment by gurupras** [comment]
Not quite.
Chrome still plays captions natively whereas plyr goes down a code path
that thinks it cannot play them natively.
I'm not sure if that's okay or if it makes more sense to have it render
natively.. (Perhaps there are some performance implications?)
I'll leave that decision to you.

**Comment by SamPotts** [comment]
The problem seems to be that Chrome decides it doesn't want to fire a cuechange event after doing it twice. Safari and Firefox have some pretty tragic bugs in this area too. It might be better to manually render them for everything which is a shame, I guess it's not important for the browser developers :-1: I'll keep digging.

**Comment by SamPotts** [comment]
Found the issue - my demo webVTT file didn't pass validation and the browsers don't seem to communicate these issues, they just fail silently - awesome. 

Found a good validator and added it to the docs: 
https://quuz.org/webvtt/

Done a quick test in the various browsers, Firefox still doesn't support the `cuechange` event - [a bug has been raised](https://bugzilla.mozilla.org/show_bug.cgi?id=1033144). I found that Safari now supports TextTrack... well, apart from ignoring the `type` attribute on the track (we set it to hidden to hide default rendering) but that's solved by hiding them with CSS.

Grab v1.5.12 and let me know how you go but I think we're all good?
